### PR TITLE
feat: Add custom baseline offset to LaTeX

### PR DESCRIPTION
**Pull request by aelkady**
## Description
Addresses #83. Adds the ability to shift LaTeX text up/down to the user's content. 

## DevQA
Initializing `InputOutputModuleView` objects in the sample projects with a custom offset should reflect appropriately when run.

### DevQA Prep
- Run `pod install`
- Test `RichTextView` to ensure that it passes (unit and UI tests)
- Navigate to the `Example` project in the root `Example` folder and run the app
- Ensure that everything in the `Example` app looks visually correct

### CODE REVIEW

File: Source/RichTextView.swift [hunk 1]
```diff
@@ -35,12 +35,13 @@ public class RichTextView: UIView {
                 textColor: UIColor = UIColor.black,
                 isSelectable: Bool = true,
                 isEditable: Bool = false,
+                latexTextBaselineOffset: Double = 0,
```
Comment (wqtophat) [inline Source/RichTextView.swift]: Would use `CGFloat` instead `Double` for offset for better consistency with iOS convention.

**Comment by wqtophat** [comment]
I feel like in most case users would simply want to vertical alignment to be top / center / baseline instead of setting a manually calculated offset value. `UILabel` provides this kind of simplicity while still allow advanced users to set the hardcoded offset in attribute of attributed string. IMO if would be great if we could offer the same level of simplicity without sacrificing flexibility.

**Comment by aelkady** [comment]
@wqtophat unfortunately we cannot use `UILabel` because it would remove the ability for us to make the `RichTextView` selectable/editable. I am fine with having a high level of flexibility for now to get this PR out of the door - we can explore a simpler way to set the offset (top/bottom/centre) if there is a strong need for it down the line.

**Comment by tophat-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Please ensure that your new files respect the existing folder structure</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">
<p>If anything other than <code>RichTextView</code> appears in the code coverage section please add it<br />
to the <code>exclude_targets</code> property of the <code>xcov.report()</code> function in the Dangerfile</p>
</td>
    </tr>
  </tbody>
</table>

## Current coverage for RichTextView.framework is `90.81%`
Files changed | - | - 
--- | --- | ---
RichTextView.swift | `86.47%` | :white_check_mark:
RichTextParser.swift | `94.31%` | :white_check_mark:
LatexParserProtocol.swift | `95.65%` | :white_check_mark:

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**